lib/client.c

```diff
@@ -620,6 +644,15 @@ snowflake_global_set_attribute(SF_GLOBAL_ATTRIBUTE type, const void *value) {
         case SF_GLOBAL_OCSP_CHECK:
             SF_OCSP_CHECK = *(sf_bool *) value;
             break;
+        case SF_GLOBAL_CLIENT_CONFIG_FILE:
+            alloc_buffer_and_copy(&CLIENT_CONFIG_FILE, value);
+            break;
+        case SF_GLOBAL_LOG_LEVEL:
```

Comment:
I would prefer SF_GLOBAL_LOG_PATH and SF_GLOBAL_LOG_LEVEL to be read only attributes since application already have enough ways to do log settings.